clients/cobol-lsp-vscode-extension/src/Worker.ts

```diff
@@ -0,0 +1,24 @@
+import { parentPort } from "worker_threads";
+import { Program } from "@code4z/analysis/lib/model/cfast";
+import { ControlFlowGraphBuilder } from "@code4z/analysis/lib/graphbuilder";
+
+export class WorkerMessage {
+  public constructor(
+    public vmCount: number,
+    public severity: number,
+    public programs: Program[],
+  ) {}
+}
```

Comment:
I suggest moving this into a separate file, so you don't end up pulling the whole worker into the other module.
```
export type WorkerMessage = {
    vmCount: number,
    severity: number,
    programs: Program[],
}
```